src/olympia/migrations/920-webext-permissions-file-fk-is-unique.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE `webext_permissions` DROP FOREIGN KEY `webext_permissions_file`;
+
+-- This will fail locally as file.id is by default signed, but unsigned on prod, etc.
+ALTER TABLE `webext_permissions` MODIFY `file_id` int(11) unsigned UNIQUE NOT NULL;
```

Comment:
you thought correctly https://github.com/mozilla/addons-server/pull/4647/commits/1be4cdf52da5b34594ea2b07e89b8d968196202e